packages/api/lambdas/ems-distribution-report.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+const log = require('@cumulus/common/log');
+const moment = require('moment');
+const { aws } = require('@cumulus/common');
+const { TaskQueue } = require('cwait');
+const { URL } = require('url');
+
+/**
+ * This class takes an S3 Server Log line and parses it for EMS Distribution Logs
+ */
+class DistributionEvent {
+  static isDistributionEvent(s3ServerLogLine) {
+    return s3ServerLogLine.includes('REST.GET.OBJECT')
+      && s3ServerLogLine.includes('x-EarthdataLoginUsername');
+  }
+
+  constructor(s3ServerLogLine) {
+    if (!DistributionEvent.isDistributionEvent(s3ServerLogLine)) {
+      throw new Error(`Invalid distribution event: ${s3ServerLogLine}`);
+    }
+
+    this.rawLine = s3ServerLogLine;
+  }
+
+  get bucket() {
```

Comment:
Can you please comment on what an expected line of the log looks like so we can see how this is being split up?